### ISSUE TITLE
enhancement: add termcap

### ISSUE THREAD

**Issue by snakebyte69** [avalonbits/vdp-gl#15]
Add the termcap Database so any terminal can be used
https://www.gnu.org/software/termutils/manual/termcap-1.3/html_mono/termcap.html

**Comment by avalonbits** [comment]
Can you be more specific?
What about adding the termcap database would make it possible for any terminal to be used?

**Comment by snakebyte69** [comment]
Yes it's a large database of terminal information for emulators to use. e.g. Kermit95, C-Kermit.
A FabGL based board could use the existing GUI but switch through a huge list of terminals.
